CMakeLists.txt

```diff
@@ -148,7 +150,14 @@ if(UNIX)
 endif(UNIX)
 
 # CMake provided find modules
-find_package(ZLIB)
+if(BUILD_STATIC)
+    find_library(ZLIB NAMES libz.a REQUIRED PATH_SUFFIXES lib lib/x86_64-linux-gnu lib64)
+    if(ZLIB)
+        message(STATUS "Found ZLIB: ${ZLIB}")
```

Comment:
Is this just a debugging statement left in the CMakeLists.txt?  There are no other message()s in CMakeLists.txt.  Perhaps this should be a FATAL_ERROR if ZLIB is not found.

Reply:
Ok, removed.